integration/test_alert_rules.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from integration import run_command
+
+ALERT_RULES_COMMAND = "code42 alert-rules"
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        f"{ALERT_RULES_COMMAND} list",
```

Comment:
`string.format` is more consistent with how we format strings in py42

Reply:
changed.